R_Friedmantest.qmd

```diff
@@ -0,0 +1,52 @@
+---
+title: "Friedman Chi-Square test using R package rstatix"
+---
+
+# **Example: Simulated dataset with 10 subjects each having results at each of the 4 time points.**
+
+Simulated datasets with continuous endpoints are generated for single-drug repeated measurements to check whether any significance exists between the responses at different time points. The p-value will indicate whether differences in response for different time points are significant.
+
+*Data source:*
+
+```{r, output = FALSE, message = FALSE}
+library(dplyr)
+library(rstatix) 
+```
+
+```{r eval=FALSE}
```

Comment:
The CAMIS repo is set up with a github action that runs the R and python code and creates the output. Therefore all the R code blocks should have eval = TRUE

```suggestion
```{r}
```